### PR TITLE
Add windows requirements into setup.py

### PR DESCRIPTION
**Pull request by magicarm22**
Was added windows requirements into setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,6 +17,9 @@
 REQUIRES = [
     'comment_parser == 1.2.3',
     'thefuzz[speedup] == 0.19.0',
```
Comment (kumekay) [inline setup.py:19]:
```suggestion
    'thefuzz == 0.19.0',
```

File: setup.py [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+    'thefuzz[speedup] == 0.19.0; sys_platform != "win32"',
```
Comment (kumekay) [inline setup.py:20]:
```suggestion
    'thefuzz[speedup] == 0.19.0; sys_platform != "win32"',
    # don't depend on python-Levenshtein on Windows, as it requires Microsoft C++ Build Tools to install
```

**Review by kumekay — CHANGES_REQUESTED** [review]
Why would we still need `requirements.txt` in this repo?

I also see that here check_copyright.py is about 100 lines shorter than in IDF. What has changed?

I suggest preserving the development history of the script from ESP-IDF. You can do it using https://github.com/newren/git-filter-repo

**Comment by kumekay** [comment]
@magicarm22 One more suggestion, add some examples to the readme

**Comment by magicarm22** [comment]
@kumekay About history - https://github.com/espressif/check-copyright/compare/feature/copy_history?diff=unified
But I don't understand, how can I rewrite history in master with PR. Could you please check?

**Pull request closed without merging**